database/initial_structure.sql

```diff
@@ -0,0 +1,37 @@
+CREATE SEQUENCE batch_id_seq
```

Comment:
If you name this `dces-drc-integration/src/main/resources/schema.sql`, I think you'll get some nice extra behaviour for free (and it won't run it by default on application startup because PostgreSQL is not an in-memory database).

See https://docs.spring.io/spring-boot/how-to/data-initialization.html#howto.data-initialization.using-basic-sql-scripts